extension/manifest.json

```diff
@@ -25,6 +25,17 @@
         "48": "images/GearHeart_Mixed_47x47.png",
         "128": "images/GearHeart_Mixed_127x127.png"
     },
+    // added a keyboard shortcut or command to open the panel
+    "commands": {
+        "_execute_browser_action": {
+            "suggested_key": {
+              "windows": "Alt+O",
+              "mac": "Command+Shift+Y",
+              "chromeos": "Ctrl+Shift+U"
```

Comment:
I understand that the suggested keyboard shortcut would not be permitted by the [chrome.commands](https://developer.chrome.com/apps/commands) api; however, could you explain the reason for the options you have chosen.

Reply:
I added this keyboard shortcut to a normal "Hello World" extension and it executed fine. In [chrome.commands](https://developer.chrome.com/apps/commands) api, Cltrl+Alt is not permitted but the keyboard shortcut I included will work fine. Can you check the api once and confirm if I'm wrong?

Reply:
The question is more, why these key combinations? Why are they different across systems?

Reply:
I just took "Alt+O" randomly and was able to open the extension panel. Also, rest two keyboard shortcuts for "mac" and "chromeos" systems are taken at random. Here, even if you take any other key shortcut for eg: "Alt+J", it should work fine. Can you please guide me if this is not the right way to do it?

Reply:
It's probably better to have a consistent keyboard shortcut that where possible is the same across platforms. That way if you use multiple systems it won't be confusing to know which shortcut to use. 

Also the issue ([GPII-4241](https://issues.gpii.net/browse/GPII-4241)) suggested a keyboard shortcut. While the suggested one won't work, you can look into creating something similar based off of it. Also you may want to do some investigation about existing shortcuts, to ensure that you minimizing conflicts.